Added intercept survey guidance article (#2278)
MicrosoftDocs/azure-docs-powershell: docs-conceptual/azps-8.2.0/intercept-survey-guidance.md

---
title: Intercept survey guidance
description: This article contains information about azure powershell intercept survey guidance.
ms.date: 08/31/2022
ms.devlang: powershell
ms.service: azure-powershell
ms.topic: conceptual
ms.custom: devx-track-azurepowershell
---

> # Intercept survey guidance

Reviewer: ```suggestion
# Azure Command line tools survey guidance
```

> The purpose of the intercept survey is to collect information about the satisfaction of Azure
> command line tools. By providing responses to the survey, you help to identify common issues and
> areas for improvement. Understanding your experiences and opinions helps to make future releases of
> Azure command line tools better for you and others.

Reviewer: ```suggestion
When using Azure PowerShell you may be invited to fill a survey to tell us about your experience. By providing responses to the survey, you help to identify common issues and areas for improvement. Understanding your experiences and opinions helps to make future releases of
Azure command line tools better for you and others.
```

## What data is collected

> The intercept survey collects anonymized feedback about your satisfaction of Azure command line
> tools. The intercept survey does not collect any private or personal data.
> 
> While we appreciate the insights this data provides, we understand not everyone wants to be prompted
> to complete a survey. You can disable being prompted to participate in surveys with the
> `Update-AzConfig` cmdlet or via an environment variable.

Reviewer: ```suggestion
The survey collects anonymized feedback about your satisfaction of Azure command line
tools and does not collect any private or personal data.

While we appreciate the insights this data provides, we understand not everyone wants to be prompted
to complete a survey. You can disable being prompted to participate in surveys with the
`Update-AzConfig` cmdlet or via an environment variable.
```
